prime-router/src/test/kotlin/serializers/CsvSerializerTests.kt

```diff
@@ -228,6 +228,67 @@ class CsvSerializerTests {
         assertThat(csv).isEqualTo(expectedCsv)
     }
 
+    @Test
+    fun `test multiple datetime formatting`() {
```

Comment:
I would like to see unit tests that test the functions you just added.  These are fine, but tend to be more on the integration side than unit test.

Reply:
Code coverage for the two new functions is 92+%

Reply:
Thank you for your advise.  I will create ticket for this and put it in nice to have.